File: fabric/src/main/java/dev/vini2003/hammer/fabric/HClient.java
package dev.vini2003.hammer.fabric;

import dev.vini2003.hammer.core.HCClient;
import net.fabricmc.api.ClientModInitializer;

public class HClient implements ClientModInitializer {
	@Override
	public void onInitializeClient() {
		HCClient.onInitializeClient();
	}
}

File: src/main/kotlin/dev/vini2003/hammer/HClient.kt
package dev.vini2003.hammer

import dev.vini2003.hammer.registry.client.HEvents
import dev.vini2003.hammer.registry.client.HCommands
import dev.vini2003.hammer.registry.client.HScreens
import net.fabricmc.api.ClientModInitializer
import net.fabricmc.loader.api.FabricLoader

object HClient : ClientModInitializer {
	override fun onInitializeClient() {
		HEvents.init()
		
		if (FabricLoader.getInstance().isDevelopmentEnvironment) {
			HCommands.init()
			HScreens.initialize()
		}
	}
}